PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ruleConfigs.json

```diff
@@ -0,0 +1,5 @@
+{
+  "StorageUploadWithoutLengthCheck": {
+    "methodsToCheck": ["uploadFromFile", "upload", "uploadWithResponse"]
```

Comment:
Did we account for https://github.com/Njeriiii/azure-tools-for-java/pull/2/files/f9be8804b6038534a8548a633b7b2511581ebac0#r1634828610

Reply:
yes -- uploadFromFile isn't necessary to flag